elasticluster/share/playbooks/roles/kubernetes-common/tasks/redhat.yml

```
@@ -0,0 +1,30 @@
+---
+
+- name: Add Kubernetes yum repo
+  yum_repository:
+    name: Kubernetes
+    description: Kubernetes
+    file: kubernetes
+    baseurl: http://yum.kubernetes.io/repos/kubernetes-el7-x86_64
+    enabled: true
+    gpgcakey: https://packages.cloud.google.com/yum/doc/yum-key.gpg
+    gpgkey: https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+    gpgcheck: true
+    state: present
+
+- name: Workaround to https://github.com/kubernetes/kubernetes/pull/33555
+  command: setenforce 0
+  ignore_errors: true
```

Should probably not ignore errors here: if `setenforce` fails, SELinux will still be in strict mode, which will cause things to go wrong later on in a less easier to debug setting.